File: Server/Toopeto/JsonPacket/ListTransferPacket.cs
using AnarSoft.Utility.Utilities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace Toopeto.JsonPacket
{
    public class ListTransferPacket : Json
    {
        [DataMember]
        public List<string> list;

        [DataMember]
        public int count;

        [DataMember]
        public int currentNumber;

        public ListTransferPacket()
            : base()
        {
            this.packetName = PacketNameEnum.listTransfer.ToString();
        }

        public override string getString()
        {
            return JsonSerializer.Serialize(this);
        }
        public static ListTransferPacket GetListTransferPacket(string jsonString)
        {
            return JsonSerializer.DeSerialize<ListTransferPacket>(jsonString);
        }

    }
}

File: ClientAndroid/src/main/java/ir/bilgisoft/toopeto/json/ListTransferPacket.java
package ir.bilgisoft.toopeto.json;

import com.google.gson.Gson;

import java.util.List;

/**
 * Created by mehrang on 20/03/2015.
 */
public class ListTransferPacket extends  Json{
    public List<String> list;
    public int count;
    public int currentNumber;

    public ListTransferPacket()
    {
        super();
        this.packetName =Enums.PacketNameEnum.listTransfer.toString();
    }

    @Override
    public  String getString()
    {
        Gson gson = new Gson();
        String json = gson.toJson(this);
        return json;
    }

    public static ListTransferPacket GetListTransferPacket(String JsonPacket)
    {
        Gson gson = new Gson();
        ListTransferPacket lp= gson.fromJson(JsonPacket,ListTransferPacket.class);
        return lp;
    }

}
